src/pages/benchmarks/index.tsx

```diff
@@ -11,22 +11,50 @@ import { useState } from "react";
 import $button from "../../components/design_system/button/Button.module.css";
 import noBenchmarks from "../../public/img/no-benchmarks.png";
 import SearchIcon from "@mui/icons-material/Search";
+import { SortDirection, SortProperty } from "@/types/global";
 
 function Benchmarks() {
   const [isPara, setIsPara] = useState(false);
-  const { data: tasks, isLoading } = trpc.para.getMyTasks.useQuery();
+
+  const [sortProperty, setSortProperty] = useState<SortProperty>("first_name");
+  const [sortDirection, setSortDirection] = useState<SortDirection>("asc");
+
+  const { data: tasksData, isLoading } = trpc.para.getMyTasks.useQuery();
 
   const handleTogglePara = () => {
     setIsPara(!isPara);
   };
 
+  const getSortedTasks = () => {
+    if (!tasksData) return [];
+
+    return [...tasksData].sort((a, b) => {
```

Comment:
Can we put all of this in a useEffect, rather than making a copy that gets sorted? I don't see a problem with changing the data that came in from the trpc